Source/WebCore/platform/graphics/SourceBufferPrivate.h

```diff
@@ -81,6 +81,7 @@ class SourceBufferPrivate
     virtual void removedFromMediaSource() = 0;
     virtual MediaPlayer::ReadyState readyState() const = 0;
     virtual void setReadyState(MediaPlayer::ReadyState) = 0;
+    WEBCORE_EXPORT virtual void clientReadyStateChanged(bool);
```

Comment:
It is a bit odd to pass a bool to clientReadyStateChanged, maybe add the name of the bool to ease readability.